### PR TITLE
Keyboard events

### LINKED ISSUE #81: Create module that fires events on key up and key down
This would allow the creation of keyboard shortcuts through module behaviors.

### PR DESCRIPTION
**Pull request by 0x4d**
Closes #81

## Describe your changes
I added a new module KeyboardEvents that add listeners for keyup and keydown that propagate events to the EventBus.
I then added behaviors to the Toolbox and Remove modules for these events.
I decided not to use the diagram-js keyboard module for this because it is to extensive and does not use our concept of behaviors.

### CODE REVIEW

File: src/core/toolbox/index.js [hunk 1]
```diff
@@ -22,6 +23,7 @@ export default {
         [ 'detach', DetachBehavior ],
         [ 'element.hover', HoverBehavior ],
         [ 'element.out', OutBehavior ],
+        [ 'keypress', KeypressBehavior ],
```
Comment (tim-kilian) [inline src/core/toolbox/index.js:26]: An Idea to improve the KeyBehaviors is to define it like this:

`[ [ 'esc' ], KeypressBehavior ],`

File: src/features/remove/index.js [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
 
 export default {
     __depends__: [
+        SelectionModule,
     ],
     __init__: [
         'remove',
     ],
     __behaviors__: [
+        [ 'keypress', KeypressBehavior ],
```
Comment (tim-kilian) [inline src/features/remove/index.js:17]: For defining multiple keyboard events for the same behavior we could just list them up like this:

```
[ [ 'del' ], KeypressBehavior ],
[ [ 'backspace' ], KeypressBehavior ],
```

**Review by tim-kilian — APPROVED** [review]
Improvment by using key events 👍

File: src/core/toolbox/index.js (hunk 1 above)
Comment (0x4d) [reply src/core/toolbox/index.js:26]: :+1:

**Pull request merged**